Rakefile

```diff
@@ -29,7 +29,7 @@ namespace "test" do
   task :prepare_testapp, :force do |_t, args|
     if args.force || !File.directory?(TESTAPP_PATH)
       FileUtils.remove_entry_secure(TESTAPP_PATH, true)
-      sh %(#{BIN} new #{TESTAPP_PATH} --skip-wizard --skip-bundle)
+      sh %(#{BIN} new #{TESTAPP_PATH} --skip-wizard --skip-bundle --api)
```

Comment:
📓 Added this to make bootstrapping easier since we don't need to test with a frontend capable rails application, this trims down the chances of configuration errors cropping up.